### PR TITLE
Details pane for total usage + small stuff

### LINKED ISSUE #12: Include a better system description on the home page
Current project description on the home page is too technical.

We need to include more statements about the mission of the project rather than technical details (e.g. reverse proxy logs).

Any suggestion is more than welcomed.

### LINKED ISSUE #38: Backend: stop container properly
When doing a `sys.exit()` in backend code (e.g. because an async task failed), this effectively stops the whole FastApi application but it does not stop the container.

We should find a way to properly exit in such a situation.

### LINKED ISSUE #55: Implement the Total Usage details page
Implement the Total Usage details page

image

image

### LINKED ISSUE #93: Frontend : fix fallback logo image
Fallback logo image (which is always used for now) should be a PNG with proper size (see #64 for details)

### PR DESCRIPTION
**Pull request by benoit74**
## Rationale

Fix #55 
Fix #12 
Fix #93

## Changes

- implement the detail pane for total usage (reusing same component for items as package popularity)
- fix the fallback image (for now, adding a dynamic one based on offspot configuration, including text, will be done later)
- add a screenshot to home page

### CODE REVIEW

**Comment by benoit74** [comment]
Hum, some commits were missing ...

File: backend/src/offspot_metrics_backend/main.py [hunk 1]
```diff
@@ -77,7 +79,11 @@ def task_stopped(self, task_name: str, task: Task[Any]) -> None:
         self.background_tasks.discard(task)
         if exc:
             logger.error(f"{task_name} has stopped anormally", exc_info=exc)
-            sys.exit(1)
+            if BackendConf.inside_docker:
+                logger.error(f"Stopping PID {os.getpid()} with SIGINT")
+                os.kill(1, signal.SIGINT)
```
Comment (rgaudin) [inline backend/src/offspot_metrics_backend/main.py:84]: This is sketchy… and misleading. Your app could be a different PID is the entrypoint doesn't replace current process but starts another one for instance. You only want to stop your app so just reuse `os.getpid()`.

File: frontend/src/components/PackageElement.vue [hunk 2]
```diff
@@ -50,9 +51,7 @@ const chartOptions = computed(() => {
       type: 'solid',
       opacity: 1,
       colors: [
-        props.inverted
-          ? '#000000'
-          : packagePopularityStore.packageColor(props.package),
+        props.inverted ? '#000000' : mainStore.getPackageColor(props.package),
```
Comment (rgaudin) [inline frontend/src/components/PackageElement.vue:54]: Not a new change but shouldn't this color be a constant somewhere?

**Review by rgaudin — APPROVED** [review]
LGTM ; please check the PID thing before merging.

I did not run the code so I'm not completely clear about the _inverted_ usage

**Comment by benoit74** [comment]
> I did not run the code so I'm not completely clear about the inverted usage

`inverted` means "black graphics on colorful background" while default (non-inverted) means "colorful graphic on white background"

image

File: frontend/src/components/PackageElement.vue (hunk 2 above)
Comment (benoit74) [reply frontend/src/components/PackageElement.vue]: Let's do it, I don't mind

**Pull request merged**